src/jvm/benchmark/org/jetbrains/kotlinx/lincheck_benchmark/AbstractLincheckBenchmark.kt

```diff
@@ -0,0 +1,82 @@
+/*
+ * Lincheck
+ *
+ * Copyright (C) 2019 - 2023 JetBrains s.r.o.
+ *
+ * This Source Code Form is subject to the terms of the
+ * Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed
+ * with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+@file:Suppress("INVISIBLE_REFERENCE", "INVISIBLE_MEMBER")
```

Comment:
To access `internal fun checkImpl(...)` from `LinChecker.kt`. 
Otherwise, code from benchmarks source set cannot access `internal` members from the `main` source set.

Reply:
Let's add `Suppress` at the places where you need it, to make the usages more clear